rules/falco_rules.yaml

```diff
@@ -2279,7 +2279,7 @@
     User management binary command run outside of container
     (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent=%proc.pname gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4])
   priority: NOTICE
-  tags: [host, users, mitre_persistence]
+  tags: [host, users, mitre_TA0003_persistence, mitre_TA0004_privilege_escalation, mitre_T1068_explotation_for_privilege_escalation]
```

Comment:
I don't think it is related to privilege escalation. The intent of the rule is to detect an attacker tried to gain some persistence in the system (create a bogus user)

Reply:
I think if an unprivileged user got to execute these commands, he escalated privileges.